### PR TITLE
Add filters documentation

### PR DESCRIPTION
**Pull request by palourde**
It adds a guide for reducing alert fatigue & reference documentation on filters.

Closes https://github.com/sensu/sensu-docs-site/issues/201
Closes https://github.com/sensu/sensu-docs-site/issues/195

### CODE REVIEW

File: content/sensu-core/2.0/guides/reduce-alert-fatigue.md [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+You can verify the proper behavior of this filter by using `sensu-backend` logs.
+Whenever an event is being handled, a log entry is added with the message
+`sending event: {...} to handler: mail`, followed by a second one with the
+message `pipelined executed event pipe handler: status=0 output=`. However, if
+the event is being discarded by our filter, a log entry with the message `event
+filtered` will appear instead.
+
+## Next steps
+
+You now know how to apply a filter to a handler and hopefully reduce alert
```
Comment (mercul3s) [inline content/sensu-core/2.0/guides/reduce-alert-fatigue.md]: This looks great as a how to guide! ✨

**Comment by mercul3s** [comment]
nice

File: content/sensu-core/2.0/guides/reduce-alert-fatigue.md [hunk 2]
```diff
@@ -0,0 +1,72 @@
...
+{{< highlight shell >}}
+sensuctl handler set-filters mail hourly
+{{< /highlight >}}
+
+### Validating the filter
+
+You can verify the proper behavior of this filter by using `sensu-backend` logs.
+Whenever an event is being handled, a log entry is added with the message
+`sending event: {...} to handler: mail`, followed by a second one with the
+message `pipelined executed event pipe handler: status=0 output=`. However, if
```
[the remark this answers is not in the document]
Comment (palourde) [reply content/sensu-core/2.0/guides/reduce-alert-fatigue.md]: We actually have an issue in sensu-go to improve these log entries (I realized they are way too complicated when writing this guide 😄) and made a reference to this document so this part gets updated when https://github.com/sensu/sensu-go/issues/1148 is done.

File: content/sensu-core/2.0/guides/reduce-alert-fatigue.md [hunk 3]
```diff
@@ -0,0 +1,72 @@
...
+Here, since we want to reduce the number of emails sent by Sensu, we will apply
+our filter to an already existing handler named `mail`.
+
+{{< highlight shell >}}
+sensuctl handler set-filters mail hourly
+{{< /highlight >}}
+
+### Validating the filter
+
+You can verify the proper behavior of this filter by using `sensu-backend` logs.
```
[the remark this answers is not in the document]
Comment (palourde) [reply content/sensu-core/2.0/guides/reduce-alert-fatigue.md]: Good point, I'll do that!

[the remark this answers is not in the document]
Comment (palourde) [reply content/sensu-core/2.0/guides/reduce-alert-fatigue.md]: Added the location in https://github.com/sensu/sensu-docs-site/pull/254, I'll link to this section once this PR is merged!

**Pull request merged**